src/pages/core/standard-library/math/integers.mdx

```diff
@@ -0,0 +1,50 @@
+---
+tags: ["core", "math"]
+---
+
+import { Callout } from "nextra/components";
+
+# Integers
+
+CosmWasm offers integer types starting at 64 bit precision, all the way up to
+512 bits.
+
+Reasoning behind wrapping the native 64 bit type is the interaction with
+JavaScript. A `u64` would normally serialize into a regular integer field in
+JSON. The issue here is that, due to representing integers in form of double
+precision floats, JavaScript can only handle up to ~48 bit numbers without
```

Comment:
```suggestion
precision floats, JavaScript can only handle up to ~53 bit numbers without
```

This is how I always got it:

```
$ node
> Math.log(Number.MAX_SAFE_INTEGER) / Math.log(2)
53
```

Reply:
🤦‍♀️ ugh.. I was mixing the bits up with pointers on AMD64. Thanks